profiles/common/modules/features/nexteuropa_webtools/nexteuropa_webtools.module

```diff
@@ -268,3 +268,15 @@ function nexteuropa_webtools_ajax_access_callback() {
   $view = views_get_view($view_name);
   return $view && $view->access($display_id);
 }
+
+/**
+ * Implements hook_custom_theme().
+ */
+function nexteuropa_webtools_ckeditor_custom_theme() {
+  if (strpos(current_path(), 'nexteuropa/webtools-ckeditor') === 0) {
+    $theme = variable_get('node_admin_theme', '0') ?
+      variable_get('admin_theme_default', 'seven') :
+      variable_get('theme_default', 'ec_resp');
```

Comment:
Before the PR 570, this code is inspired by "nexteuropa_token_ckeditor_custom_theme" where  we had: 
`if (strpos(current_path(), 'nexteuropa/token-ckeditor') === 0) {
    return 'seven';
  }`

The use of this value `variable_get('admin_theme_default', 'seven')` does not bother me so much because it is commonly used in all sites but `variable_get('theme_default', 'ec_resp')` bothers me because we cannot be sure that the default theme is compatible with the feature or Am I wrong?